### PR TITLE
Fix the pull member up failure

### PR DESCRIPTION
**Pull request by Cosifne**
Related [issue](https://github.com/dotnet/roslyn/issues/34268)
So, null pointer exception happens when you try to make a property abstract like
```
abstract class B { }

class D : B
{
    int $$X => 7;
}
```
If try to make it abstract via the pull member up dialog, it will fail.

Reason:
It is due to CodeGenerationSymbolFactory.CreatePropertySymbol() takes SetMethod and GetMethod and the default value are both null.

I also add two additional tests to cover the cases which makes events abstract.

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs [hunk 1]
```diff
@@ -2151,6 +2152,120 @@ public class Testclass2 : ITest
...
+        private static event EventHandler Event1, Eve[||]nt3, Event4;
+    }
+}";
+            var expected = @"
+using System;
+namespace PushUpTest
+{
+    public abstract class Base2
+    {
+        private static abstract event EventHandler Event3;
```
Comment (genlu) [inline src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs]: Hmm, abstract member can't be private or static. Are we intentionally breaking the code so the user will notice the error and fix them manually? Should we consider doing something automatically by default? like changing the accessibility to `protected  internal` and make it a non-static member.

**Comment by genlu** [comment]
@vatsalyaagrawal @jinujoseph IS this targeting 16.1 preview 2?

**Comment by jinujoseph** [comment]
Master should be fine

**Comment by genlu** [comment]
@Cosifne Could you please confirm that the scenario I commented above which leaves the code in a broken state after refactoring is by-design? Otherwise, we need to file a bug and possibly go through design meeting for that. Thanks

**Pull request merged**